src/core/Clipboard.cpp

```diff
@@ -92,5 +158,92 @@ namespace lmms::Clipboard
 		return( QString::fromUtf8( mimeData->data( mimeType( MimeType::StringPair ) ) ).section( ':', 1, -1 ) );
 	}
 
+	std::pair<QString, QString> decodeMimeData(const QMimeData* mimeData)
+	{
+		const QList<QUrl> urls = mimeData->urls();
+
+		QString type{"missing_type"};
+		QString value;
```

Comment:
Is `value` initialized? If not, then it is possible to return it uninitialized and zero-initialization should be used.

```suggestion
		QString value{};
```